apps/webapp/app/services/endpointApi.server.ts

```diff
@@ -254,26 +254,34 @@ export class EndpointApi {
     return HttpSourceResponseSchema.parse(anyBody);
   }
 
-  async deliverWebhookRequest(options: {
+  // TODO: ensure backward compat
+  async deliverWebhookRequests(options: {
```

Comment:
Quick comment so I don't forget to do this

Reply:
Will add batched delivery as a new action instead so we can support both more easily. Just a refactor, the mechanics won't change.